.storybook/config.tsx

```diff
@@ -12,6 +12,7 @@ import { ThemeProvider } from "styled-components"
 import Flex from "../src/components/templates/flex"
```

Comment:
The change at this file are irrelevant with the this issue. 
This change fixes a bug at storybook where labels does not change colour on dark theme.